docs/src/main/asciidoc/validation.adoc

```diff
@@ -411,6 +411,44 @@ The scope you choose for your `ConstraintValidator` bean is important:
 - If the `ConstraintValidator` bean implements the `initialize(A constraintAnnotation)` method and depends on the state of the constraint annotation, use the `@Dependent` scope to make sure each annotation context has a separate and properly configured instance of the `ConstraintValidator` bean.
 ====
 
+When injecting beans relying on the runtime configuration, use `@Inject Instance<..>`. Since constraints are initialised
```

Comment:
Let's use US English given the method is called `initialize()`:
```suggestion
When injecting beans relying on the runtime configuration, use `@Inject Instance<..>`. Since constraints are initialized
```